### DISCUSSION TITLE
redis pluggin not working

### DISCUSSION THREAD

**Discussion by manzurshaikh** [Kong/kong#8271] [Help] [answered]
Question: getting below error while configuring redis pluggin

Install Plugin - ok
` sh -c "git clone https://github.com/globocom/kong-plugin-proxy-cache /tmp/kong-plugin-proxy-cache && cd /tmp/kong-plugin-proxy-cache && luarocks make *.rockspec"
Cloning into '/tmp/kong-plugin-proxy-cache'...
remote: Enumerating objects: 671, done.
remote: Total 671 (delta 0), reused 0 (delta 0), pack-reused 671
Receiving objects: 100% (671/671), 86.62 KiB | 6.66 MiB/s, done.
Resolving deltas: 100% (450/450), done.

kong-plugin-proxy-cache 2.0.0-1 is now installed in /usr/local (license: MIT)`

Enable Plugin - error
`curl -X POST  http://kong:8001/plugins/ --data "name=proxy-cache"  --data "config.response_code=200" --data "config.request_method=GET" --data "config.request_method=HEAD" --data "config.
content_type=text/plain" --data "config.content_type=application/json" --data "config.cache_ttl=300" --data "config.strategy=memory"
{"message":"UNIQUE violation detected on '{consumer=null,name=\"proxy-cache\",route=null,service=null}'","fields":{"service":null,"name":"proxy-cache","consumer":null,"route":null},"code":5,"

`

`curl -X POST http://kong:8001/services/debug-upstream/plugins --data "name=proxy-cache" --data "config.cache_ttl=300" --data "config.redis.host=redis-host-aws.rds..com"
{"message":"Not found"}/ #`

Please guide me through first time configuring Kong using Docker

**Comment by locao** [comment, marked as answer]
Hi @manzurshaikh! As pointed out in #8276, you are using a forked version of proxy-cache plugin, so we cannot assist with it.
